pkg/networkpath/traceroute/utils.go

```diff
@@ -0,0 +1,246 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
🥜 I'm wondering if there is a better file name for `RunTraceroute` function. Maybe `traceroute_runner.go`.

`utils.go` feels fine for small utility functions, but this is a bit more than that I think.